packages/gibsonify_api/lib/src/models/recipe_item.dart

```diff
@@ -74,16 +87,25 @@ class Recipe extends Equatable {
         recipeVolume,
         ingredients,
         probes,
+        probesChecked,
+        probesStandard,
         saved,
       ];
 }
 
 List<Ingredient> _jsonDecodeIngredients(jsonEncodedIngredients) {
   List<dynamic> partiallyDecodedIngredients =
       jsonDecode(jsonEncodedIngredients);
-  List<Ingredient> fullyDecodedFoodItems = List<Ingredient>.from(
+  List<Ingredient> fullyDecodedIngredients = List<Ingredient>.from(
       partiallyDecodedIngredients.map((x) => Ingredient.fromJson(x)));
-  return fullyDecodedFoodItems;
```

Comment:
What is this change from FoodItems to Ingredients?

Reply:
It was a wrongly named variable